releasenotes/notes/process-agent-misconfig-logging-476d5463f694d57d.yaml

```diff
@@ -0,0 +1,4 @@
+---
+enhancements:
+  - |
+    Logs a warning for misconfig check when hidepid=2 for the process agent.
```

Comment:
Is this external-facing? It's quite opaque. I don't know what misconfig is, or what it means for hidepid to equal 2.

Reply:
Ah, I just read your PR comment. In that case, perhaps a clearer release note could be: "Logs a warning when the process agent cannot read other processes due to possible misconfiguration."